sdk/template/template-dpg/src/src/WidgetServiceClient.ts

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
are these changes intended to be part of this PR?

Reply:
Hi Jeff, the changes may fix the build failure discussed [here](https://teams.microsoft.com/l/message/19:f6d52ac6465c40ea80dc86b8be3825aa@thread.skype/1701062380066?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1700762445670&teamName=Azure%20SDK&channelName=General&createdTime=1701062380066). 
Could we keep the changes since this is a feature branch for `ChatSDK`?

Reply:
oh! yes I didn't see this was a feature branch, it's fine :)